diff --git a/app/src/main/java/org/b3log/siyuan/BootActivity.java b/app/src/main/java/org/b3log/siyuan/BootActivity.java
index d750c20..42b977e 100644
--- a/app/src/main/java/org/b3log/siyuan/BootActivity.java
+++ b/app/src/main/java/org/b3log/siyuan/BootActivity.java
@@ -60,7 +60,7 @@ public class BootActivity extends AppCompatActivity {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_boot);
 
-        BarUtils.setNavBarVisibility(this, false);
+        // BarUtils.setNavBarVisibility(this, false);
 
         final String channel = Utils.getChannel(this.getPackageManager());
         final Set<String> showAgreementChannels = new HashSet<>();
diff --git a/app/src/main/java/org/b3log/siyuan/JSAndroid.java b/app/src/main/java/org/b3log/siyuan/JSAndroid.java
index ed10f8f..adb889b 100644
--- a/app/src/main/java/org/b3log/siyuan/JSAndroid.java
+++ b/app/src/main/java/org/b3log/siyuan/JSAndroid.java
@@ -228,7 +228,7 @@ public final class JSAndroid {
 
         activity.runOnUiThread(() -> {
             UltimateBarX.statusBarOnly(activity).transparent().light(appearanceMode == 0).color(parseColor(color)).apply();
-            BarUtils.setNavBarVisibility(activity, false);
+            // BarUtils.setNavBarVisibility(activity, false);
         });
     }
 
diff --git a/app/src/main/java/org/b3log/siyuan/MainActivity.java b/app/src/main/java/org/b3log/siyuan/MainActivity.java
index 1cfa545..1025292 100644
--- a/app/src/main/java/org/b3log/siyuan/MainActivity.java
+++ b/app/src/main/java/org/b3log/siyuan/MainActivity.java
@@ -172,7 +172,7 @@ public class MainActivity extends AppCompatActivity implements com.blankj.utilco
             ((ViewGroup) webView.getParent()).setPadding(0, UltimateBarX.getStatusBarHeight(), 0, 0);
         }
 
-        BarUtils.setNavBarVisibility(this, false);
+        // BarUtils.setNavBarVisibility(this, false);
 
         // Fix https://github.com/siyuan-note/siyuan/issues/9726
         AndroidBug5497Workaround.assistActivity(this);
diff --git a/app/src/main/java/org/b3log/siyuan/ShortcutActivity.java b/app/src/main/java/org/b3log/siyuan/ShortcutActivity.java
index aa077f3..6d042be 100644
--- a/app/src/main/java/org/b3log/siyuan/ShortcutActivity.java
+++ b/app/src/main/java/org/b3log/siyuan/ShortcutActivity.java
@@ -65,9 +65,9 @@ public class ShortcutActivity extends AppCompatActivity {
 
         final EditText input = findViewById(R.id.full_screen_input);
         UltimateBarX.statusBarOnly(this).transparent().apply();
-        BarUtils.setNavBarVisibility(this, false);
+        // BarUtils.setNavBarVisibility(this, false);
         ((ViewGroup) input.getParent()).setPadding(0, UltimateBarX.getStatusBarHeight(), 0, 0);
-        BarUtils.setNavBarVisibility(this, false);
+        // BarUtils.setNavBarVisibility(this, false);
 
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
             // Full screen display in landscape mode on Android https://github.com/siyuan-note/siyuan/issues/14448
